web/src/components/auth/SignUpSnackbar.test.tsx

```diff
@@ -61,21 +61,21 @@ describe("SignUpSnackbar", () => {
   it("does not show registration alert when registrationAlertIsVisible is false", () => {
     setupHookWithAuth(IsAuthenticated.FALSE, false);
     expect(
-      screen.queryByText(markdownToText(Config.navigationDefaults.guestAlertTitle))
+      screen.queryByText(markdownToText(Config.navigationDefaults.guestAlertTitle)),
     ).not.toBeInTheDocument();
   });
 
   it("does not show registration alert when user is authenticated", () => {
     setupHookWithAuth(IsAuthenticated.TRUE, false);
     expect(
-      screen.queryByText(markdownToText(Config.navigationDefaults.guestAlertTitle))
+      screen.queryByText(markdownToText(Config.navigationDefaults.guestAlertTitle)),
     ).not.toBeInTheDocument();
   });
 
   it("does not show registration alert when user is unknown", () => {
     setupHookWithAuth(IsAuthenticated.UNKNOWN, false);
     expect(
-      screen.queryByText(markdownToText(Config.navigationDefaults.guestAlertTitle))
+      screen.queryByText(markdownToText(Config.navigationDefaults.guestAlertTitle)),
```

Comment:
I actually really don't love the way it's just throwing these commas into every place possible. If there's some way we can set this to be a little more discerning I would love that.

This is a styling gripe at the end of the day.